packages/components/src/summary/style.scss

```diff
@@ -279,7 +279,9 @@ $inner-border: $core-grey-light-500;
 }
 
 .woocommerce-summary__item {
-	display: block;
+	display: flex;
+    flex-direction: column;
```

Comment:
These lines need tab instead of spaces – I see I have the same in my own working copy, due to the copy/paste.